sdk/typescript/build.gradle

```diff
@@ -64,6 +68,16 @@ task build(type: Exec, dependsOn: [install, copyABI]) {
     outputs.dir("build")
 }
 
+
+task link(type: Exec, dependsOn: [build]) {
+    executable 'npm'
+    args 'link'
```

Comment:
I still don't love using global commands (this will install a global symlink on my system for the package that is built from source here). Especially if I don't know what's happening, it can lead to weird issues. `npm link` can be a useful thing for a dev, but I'm worried about doing it transparently on their system.

Can't we find a way to contain the different build patterns within the build folder, without any global changes or in-source changes?